engine/src/audio/CodecRegistry.cpp

```diff
@@ -58,22 +58,22 @@ namespace Audio {
             nameCodec[codec->getName()] = codec;
 
             // static initializer ordering issue between boost logger and other code?
-            std::cerr << "Registering codec " << codec->getName().c_str();
+            BOOST_LOG_TRIVIAL(trace) << "Registering codec " << codec->getName().c_str();
```

Comment:
@BenjamenMeyer @ministerofinformation This code may work in practice -- on all platforms, even -- but technically, it's incorrect. Even on Linux, I see this output before main() has had a chance to initialize Boost logging. That's not supposed to happen.

I don't think we need to spend any more time on it in this PR. This is fine for now. I think I'll file a follow-up issue though. The solution would be either to A) use cerr here; or B) put the first Boost logging init function in its own static initializer block, that (hopefully) runs before this one.

Reply:
This code definitely runs before main. Reverting to what we had pre-PR and filing a new issue does help to split the issues of Windows Compilation (which I originally _thought_ this was part of, but turns out to be distinct from) and initialization order dependencies for logging.